SOX18.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+from abc import ABC, abstractmethod
```

Comment:
```suggestion
from abc import ABC, abstractmethod
from __future__ import annotations
```
это фикс кавычек (см. дальше по коду)